### PR TITLE
Add v4.2.4 and v4.5.2 releases

### PR DESCRIPTION
**Pull request by davidcassany**
Adds the release notes for v4.5.2 and v4.2.4

### CODE REVIEW

**Comment by davidcassany** [comment]
@r0ckarong I'd appreciate a quick review even this won't be definitive until I get the whole report from QA so I can add any potential issue or remark found during QA testing.

File: adoc/release-4.5.2.adoc [hunk 1]
```diff
@@ -2,21 +2,49 @@
...
+* link:https://bugzilla.suse.com/show_bug.cgi?id=1177361[bsc#1177361] [skuba] - VUL-0: CVE-2020-8030: skuba: Insecure /tmp usage when joining node to cluster
+* link:https://bugzilla.suse.com/show_bug.cgi?id=1177362[bsc#1177362] [skuba] - VUL-0: CVE-2020-8029: skuba: Insecure handling of private key
+* link:https://bugzilla.suse.com/show_bug.cgi?id=1177661[bsc#1177661] [kubernetes-1.18] - VUL-0: CVE-2020-8565: kubernetes: Incomplete fix for CVE-2019-11250 allows for token leak in logs when logLevel >= 9
+* link:https://bugzilla.suse.com/show_bug.cgi?id=1177660[bsc#1177660] [kubernetes-1.18] - VUL-0: CVE-2020-8564: kubernetes: Docker config secrets leaked when file is malformed and loglevel >= 4
 
 [[docs-changes-452]]
 === Documentation Changes
 
-
+* None
```
Comment (davidcassany) [inline adoc/release-4.5.2.adoc]: I guess I should add here the helm3 changes, however it is not clear to me how to reference them as I see that it is quite spread over different places in docs.

Comment (r0ckarong) [reply adoc/release-4.5.2.adoc]: Added some commits to add the docs change including a link. The migration is the most important part for the RN. The rest is only relevant in context with the respective document sections.

**Review by r0ckarong — APPROVED** [review]

**Comment by davidcassany** [comment]
@r0ckarong we are ready on it. No new issues have been found by QA so there is no need to modify the release-notes

**Pull request merged**